src/pages/guides/eqp/v1/index.md

```diff
@@ -18,8 +18,8 @@ The **Marketplace Developer Portal** provides you the capability to do the follo
 
 There are two separate **Marketplace Developer Portal** environments where the Marketplace EQP API can be used:
 
--  Production - [https://developer.magento.com][1]
--  Sandbox - [https://developer-stg.magento.com][2]
+-  Production - [https://commercedeveloper.adobe.com][1]
+-  Sandbox - [https://commercedeveloper-sandbox.adobe.com][2]
 
 Use your same login information for the production **Marketplace Developer Portal** to also log into the **Sandbox Marketplace Developer Portal**. Learn more about the [sandbox](sandbox.md) environment.
 
```

Comment:
Section "API availability" - we need to update the list of partners who are eligible. Please remove the table and add the following list: 

* Adobe Platinum Technology Partner 
* Adobe Gold Technology Partner
* Adobe Silver Technology Partner
* Adobe Platinum Solution Partner 
* Adobe Gold Solution Partner
* Adobe Silver Solution Partner 
* Adobe Bronze Solution Partner